cmd/install.go

```diff
@@ -108,19 +108,57 @@ func getUserList(sysUsers v1beta1.SystemUser) []string {
 }
 
 func getCmdArgs(role string) []string {
+	params := make(map[string]string)
+
+	params["data-dir"] = dataDir
+
 	var args []string
 
 	if role == "server" {
 		args = append(args, "server")
+
+		//server params
+		if serverWorkerProfile != "default" {
+			params["profile"] = serverWorkerProfile
+		}
+
+		if enableWorker {
+			params["enable-worker"] = ""
+		}
+
+		params["token-file"] = serverToken
```

Comment:
I don't see where this `serverToken` comes from

Reply:
it comes from https://github.com/k0sproject/k0s/blob/main/cmd/server.go#L59

Reply:
How does it appear in `cmd/install.go`'s `getCmdArgs` function scope?

Reply:
![Uploading image.png…]()

Reply:
![image](https://user-images.githubusercontent.com/224971/103540378-46893e00-4ea2-11eb-858b-176f5ef1e3b0.png)